geopyspark/geotrellis/catalog.py

```diff
@@ -365,14 +369,14 @@ def query(layer_type,
 
     else:
         if time_intervals:
-            time_intervals = [time.isoformat() for time in time_intervals]
+            time_intervals = [time.isoformat() + "Z" for time in time_intervals]
```

Comment:
Good catch. I'm not sure how let that one slip past me. I'll put of a fix for it in a couple of minutes.